docs/rules/newline-after-var.md

```diff
@@ -19,111 +19,122 @@ The problem is when these developers work together in a project. This rule enfor
 
 ## Options
 
-This rule takes one option, a string, which can be:
+This rule has a string option:
 
-* `"always"` enforces empty newlines after `var`, `let` or `const` (default)
-* `"never"` disallows empty newlines after `var`, `let` or `const`
+* `"always"` (default) requires an empty line after `var`, `let`, or `const`
 
-The following patterns are considered problems:
+  Comments on a line directly after var statements are treated like additional var statements.
+
+* `"never"` disallows empty lines after `var`, `let`, or `const`
+
+### always
+
+Examples of **incorrect** code for this rule with the default `"always"` option:
 
 ```js
 /*eslint newline-after-var: ["error", "always"]*/
+/*eslint-env es6*/
 
 var greet = "hello,",
     name = "world";
 console.log(greet, name);
-```
-
-```js
-/*eslint newline-after-var: ["error", "never"]*/
-/*eslint-env es6*/
 
 let greet = "hello,",
     name = "world";
-
 console.log(greet, name);
-```
-
-```js
-/*eslint newline-after-var: "error"*/  // defaults to always
-/*eslint-env es6*/
 
 var greet = "hello,";
 const NAME = "world";
 console.log(greet, NAME);
+
+var greet = "hello,";
+var name = "world";
+// var name = require("world");
+console.log(greet, name);
 ```
 
-The following patterns are not considered problems:
+Examples of **correct** code for this rule with the default `"always"` option:
 
 ```js
 /*eslint newline-after-var: ["error", "always"]*/
+/*eslint-env es6*/
 
 var greet = "hello,",
     name = "world";
 
 console.log(greet, name);
-```
-
-```js
-/*eslint newline-after-var: ["error", "never"]*/
-/*eslint-env es6*/
 
 let greet = "hello,",
     name = "world";
-console.log(greet, name);
-```
 
-```js
-/*eslint newline-after-var: "error"*/  // defaults to always
-/*eslint-env es6*/
+console.log(greet, name);
 
 var greet = "hello,";
 const NAME = "world";
 
 console.log(greet, NAME);
+
+var greet = "hello,";
+var name = "world";
+// var name = require("world");
+
+console.log(greet, name);
 ```
 
-Note: in `"always"` mode, comments on a line directly after var statements are treated like additional var statements.
-That is, they do not require a blank line between themselves and the var statements above, but do require a blank line after them.
+### never
 
-The following patterns are considered problems:
+Examples of **incorrect** code for this rule with the default `"never"` option:
 
 ```js
-/*eslint newline-after-var: ["error", "always"]*/
+/*eslint newline-after-var: ["error", "never"]*/
+/*eslint-env es6*/
 
-var greet = "hello,";
-var name = "world";
-// var name = require("world");
-console.log(greet, name);
+var greet = "hello,",
+    name = "world";
 
+console.log(greet, name);
 
-/*eslint-disable camelcase*/
-var greet = "hello,";
-var target_name = "world";
-/*eslint-enable camelcase*/
+let greet = "hello,",
+    name = "world";
 console.log(greet, name);
```

Comment:
Is it possible that this sub-example is reversed?
* does not have empty line here in **incorrect** code for this rule with the `"never"` option
* has empty line in **correct** code